commit a86359a79cc19a8bc8814e12ad778fc7cbfa6f0b
Author: Ben Webb <ben@salilab.org>
Date:   Mon Aug 18 10:31:11 2025 -0700

    Don't require Boost::System library
    
    We don't explicitly link against boost_system,
    and it became header-only in Boost 1.89. So only
    check for the libraries we actually use directly,
    such as Boost::FileSystem.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c2a05fef..016c426f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,7 +56,7 @@ endif(POLICY CMP0167)
 
 # C++11 can use the boost version
 set(Boost_NO_BOOST_CMAKE ON)
-find_package(Boost COMPONENTS system filesystem thread program_options iostreams
+find_package(Boost COMPONENTS filesystem thread program_options iostreams
   REQUIRED)
 
 include(FindHDF5)
